lms/static/coffee/fixtures/calculator.html

```diff
@@ -1,13 +1,13 @@
 <ul>
   <li class="calc-main">
-    <a href="#" class="calc" title="Open Calculator" role="button" aria-controls="calculator_wrapper" aria-expanded="false">Open Calculator</a>
+    <button class="calc" title="${_('Open Calculator')" aria-controls="calculator_wrapper" aria-expanded="false">${_('Open Calculator')</button>
```

Comment:
We shouldn't need both though. If the text is offscreen, then can't we just have the title?